travertino/src/travertino/style.py

```diff
@@ -5,12 +5,19 @@
 from contextlib import contextmanager
 from warnings import filterwarnings, warn
 
+from .compat import _toga_lt_5
 from .properties.shorthand import directional_property
 from .properties.validated import validated_property
 
 # Make sure deprecation warnings are shown by default
 filterwarnings("default", category=DeprecationWarning)
 
+_DEPRECATION_MSG = (
+    "You're probably seeing this because you've updated Travertino to 0.5.x but are "
+    "still using Toga <= 0.4.8; to fix, either update Toga to >= 0.5.0, or pin "
```

Comment:
"still" is mildly prejorative here.
```suggestion
    "using Toga <= 0.4.8; to fix, either update Toga to >= 0.5.0, or pin "
```